essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -1,154 +1,208 @@
+
+Once you're ready, start looking for a PR to work on. If you need help getting started, look at AppFlowy's <a href="https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions">Documentation</a>.
+
+### Finding an Issue to Work On
+
+**Step 1:** Go to [AppFlowy's repository](https://github.com/orgs/AppFlowy-IO/repositories) section on GitHub. This is a list of all of AppFlowy's repositories open for contributions. For the purposes of this article, we shall be looking for *Issues* within the main [AppFlowy](https://github.com/AppFlowy-IO/AppFlowy) repository.
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/715c8a61-af5f-4856-840f-11f8604e765c)
-  *Appflowy Github Page - With all open-source repositories that can be worked on*
-  
-  **Step 2:** Go to the <b>Issues</b> section of the repository. It should look something like this:
-  
+  *AppFlowy Github Page - With all open-source repositories that can be worked on*
+
+**Step 2:** Go to the *Issues* section of the repository. It should look something like this:
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/14fdc0e3-1517-40da-9576-4646b0516b36)
   *Issues Section*
-  
-  **Step 3:** Go to the label subsection, and type in <b>Good First Issue for Devs</b>.
-  
+
+ **Step 3:** Go to the label subsection, and type in "Good First Issue for Devs"
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/f5dedcd1-5f08-483d-9898-6372022060b3)
   *Label Subsections Within Issues*
-  
-  **Step 4:** Here, you can go through a list of Issues that can be worked on. They're fairly straightforward and can help you get accustomed to the  development environment as well as the codebase. Feel free to ask anyone from the Appflowy team for suggestions or to assign you to an Issue.
-  
+
+**Step 4:** Here, you can go through a list of *Issues* that can be worked on. They're fairly straightforward and can help you get accustomed to the development environment as well as the codebase. 
+
+Feel free to ask anyone from the AppFlowy team for suggestions or to assign you to an Issue.
+
   ![image](https://github.com/JRS296/AppFlowy-Docs/assets/70965472/e1022103-e98e-42b5-a43d-03f5c5b08f04)
   *Good First Issue for Devs*
-  
-  <p> Issues are generally categorised into either <b>Feature [FR]</b> or <b>Bugs [Bug]</b>. You can take a look at the description provided for the given issue. Send a message down in the comments section to get the issue assigned to yourself. Once that's done, you are ready to begin coding!</p>
-  
+
+Issues are generally categorized into either a <b>Feature [FR]</b> or a <b>Bug [Bug]</b>. You can take a look at the description provided for the given issue. 
+
+Send a message down in the comments section to get the issue assigned to yourself. Once that's done, you are ready to begin coding!
```

Comment:
```suggestion
Send a message down in the comments section to get the issue assigned to yourself. Once that's done, you are ready to begin coding!

Note: 
- Don't worry about being assigned before starting work on it. Even if you aren't assigned, all pull requests that solve a GitHub issue is appreciated.
- However, please do understand that even though you can work on issues that haven't been assigned to you, there is an increased possibility that this will lead to merge conflicts should it end up being assigned to someone else. 
- You are also free to suggest improvements on an issue assigned to someone else. 
```

Reply:
I hope this will do as far as the note is concerned. I feel it's better that we ask newcomers to message a member from the core team before getting started. I believe that is a part of the workflow for solving issues according to this article, so we ought to remain consistent.
As always, feel free to share your opinion on the same. @annieappflowy @egp415 @Xazin

Reply:
It should be "all pull requests that solve a GitHub issue are appreciated." (not "is appreciated")

I think it's worth discussing whether it's a good idea to suggest inexperienced developers / newcomers to the project start contributing before getting the go ahead per the process you have outlined — I can see it becoming more informal once someone has become an established contributor, but this is not necessarily something you  would want to even discuss within the context of this document.

Reply:
Made the change 👍 

Yes, we could discuss it further. @annieappflowy and @Xazin your thoughts?

Reply:
I'm not sure if there is a demerit. Of course if someone is missing context or is inexperienced, that means the PR could be wrong. But asking "Please assign me" does not change that.

If a PR needs changes, or is wrong, then that is that. It can happen.

Opening a PR is not really destructive, I don't see the problem.